driver/network_driver.go

```diff
@@ -65,13 +69,29 @@ func NewNetworkDriver(client *datastoreClient.Client) network.Driver {
 		ifPrefix:         IFPrefix,
 		DummyIPV4Nexthop: "169.254.1.1",
 
+		vethMTU: 1500,
```

Comment:
@ti-mo would it be safer to have the default as `0` (i.e. the current behaviour) instead of `1500`?  Unless we're sure that the kernel's default in all circumstances is `1500` then changing this default might break systems that rely on the kernel's behaviour.  WDYT?

Reply:
Crap, I was worried about this. I couldn't find the default anywhere, but it happened to default to 1500 on the interface types (veth) we create. This will definitely not the case for other overlay interface types.

There does not seem to be a canonical default defined per interface type in the `netlink` package either, so, logically, this is up to the kernel to infer when it receives a zero MTU value in a netlink message. I'll just remove this so `vethMTU` is always 0 unless overridden.

Thanks!

Reply:
Yeah, lets have it default to "what we do now", _or_ to a smaller value:  with IPIP enabled, it should probably be 1460 to allow for the IPIP header.